opengrok-indexer/src/test/java/org/opengrok/indexer/history/BoundaryChangesetsTest.java

```diff
@@ -74,6 +78,16 @@ void testInvalidMaxCount(int maxCount) {
         assertThrows(RuntimeException.class, () -> new BoundaryChangesets(gitSpyRepository));
     }
 
+    @Test
+    void testMaxCountConfiguration() {
+        int maxCount = 42 * new Random().nextInt(100) + 1;
```

Comment:
Why random? I think it's better to avoid random stuff in tests.

Reply:
just wanted to have non default value. Agreed that randomness in tests should be avoided, unless it is seeded to make it reproducible.